### DISCUSSION TITLE
Thread 1.4 git tag

### DISCUSSION THREAD

**Discussion by MichalPoterek** [openthread/openthread#10944] [Q&A]
Hi I cannot find any information about git tag for Thread 1.4
Would be appriciated for this information ;)

**Comment by jwhui** [comment]
The existing tags in the openthread/openthread repo are simply to indicate what git commit is used for a given Thread Test Harness release.

We do not yet have a tag for Thread 1.4. For Thread 1.4, I would suggest using the latest in main branch.

**Reply by tleacmcsa** [reply]
@jwhui , We are looking to stabilize our product for 1.4 certification as a border router.  Since the certification program is open for 1.4 (I think), does anyone have a feel for what others are using or what is recommended for this?  Some snapshot of the latest main branch doesnt feel right.  Are we still too early?
